README.md

```diff
@@ -11,11 +11,16 @@ AI-powered natural language end-to-end testing framework.
 
 ## Installation
 ```bash
-npm install @antiwork/shortest
+npm install -D @antiwork/shortest
 # or
-pnpm add @antiwork/shortest
+pnpm add -D @antiwork/shortest
 # or
-yarn add @antiwork/shortest
+yarn add -D @antiwork/shortest
+```
+
+Add `.shortest/` to your `.gitignore`:
```

Comment:
```suggestion
Add `.shortest/` to your `.gitignore` (where Shortest stores screenshots of each test run):
```

"that .shortest/ is not needed to be tracked" isn't really helpful, that's just explaining what the code does. I think this would be better.

Reply:
Looking at the open PRs right now .shortest/ will soon store more than screenshots. Maybe we can say something more generic like `where shortest stores test related data`!?

Reply:
We can update the readme in those PRs themselves. Each PR should be atomic.